docs/source/user/interface.rst

```diff
@@ -116,11 +116,13 @@ The browser’s native context menu can be accessed by holding down
 Keyboard Shortcuts
 ~~~~~~~~~~~~~~~~~~
 
-As in the classic Notebook, you can navigate the user interface through keyboard shortcuts.
-To find these shortcuts, go to the File, select the Settings menu option, then select Keyboard Shortcuts.
-With Keyboard Shortcuts in the settings editor, you can see a full list of keyboard shortcuts, and customize them as well.
+As in the classic Notebook, you can navigate the user interface through keyboard
+shortcuts. You can find and customize the current list of keyboard shortcuts by
+selecting the Advanced Settings Editor item in the Settings menu, then selecting
+Keyboard Shortcuts in the settings tab.
 
 [screenshot]
 
-You can also navigate JupyterLab using vim, emacs, and Sublime Text key-maps.
-Customize your shortcuts with these settings, by selecting Settings, Text Editor Key Map, and selecting your preferred key-map.
+You can also customize the text editor to use vim, emacs, or Sublime Text
+keyboard shortcuts by using the Text Editor Key Map submenu in the Settings
+menu.
```

Comment:
We should probably follow this with a screenshot as well

Reply:
Yes!